executor/common_linux.h

```diff
@@ -1557,6 +1557,82 @@ static long syz_io_uring_submit(volatile long a0, volatile long a1, volatile lon
 
 #endif
 
+#if SYZ_EXECUTOR || __NR_syz_usbip_server_init
+
+#include <errno.h>
+#include <fcntl.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <sys/socket.h>
+#include <unistd.h>
+
+static int proc_id;
+
+static long syz_usbip_server_init(void)
+{
+	int fd;
+	int rc;
+	int socket_pair[2];
+	char buffer[100];
+	int server_fd, client_fd;
+	char path[100];
+
+	rc = socketpair(AF_UNIX, SOCK_STREAM, 0, socket_pair);
+	if (rc < 0) {
+		fail("syz_usbip_server_init : socketpair failed: %d\n", rc);
+	}
+
+	client_fd = socket_pair[0];
+	server_fd = socket_pair[1];
+
+	sprintf(buffer, "%s %d %s", "0", client_fd, "65537 3");
+	sprintf(path, "%s%d%s", "/sys/devices/platform/vhci_hcd.", proc_id, "/attach");
+
+	__atomic_fetch_add(&proc_id, 1, __ATOMIC_RELAXED);
```

Comment:
This isn't right, we want to use the already existing `proc_id` in `vhci_hcd.%d`, and we want to use a new variable that needs to defined and incremented as the port number when we write into the `attach` file.

Reply:
But isn't port number actually the same value as proc_id here? Why do we need to define a new variable?

Reply:
`proc_id` is the number of the fuzzing process within the VM. We allocate one `vhci_hcd` per such process. Then each of those processes might have multiple devices created with `syz_usbip_server_connect`. For each of those devices we use a dedicated port.